site/content/docs/reference/cli.md

```diff
@@ -43,6 +43,9 @@ pinniped get kubeconfig [flags]
 - `--concierge-authenticator-type string`:
 
   Concierge authenticator type (e.g., 'webhook', 'jwt') (default: autodiscover)
+- `--concierge-mode`:
```

Comment:
This is an aside...is this file generated? I feel like we could generate this file from our `cmd/pinniped/cmd` code to make sure it stays in sync. We don't really have a good generation process to hook it into, I suppose.

I asked mostly because I think we are missing some command line options (not necessarily related to impersonation docs, e.g., `--concierge-ca-bundle`).

Reply:
Yeah, it seems like maybe we could do something with the `pinniped get kubeconfig --help` output.

Reply:
I just did this manually. I guess we could write a little script for it but it was pretty easy to do by hand, I just multiselected and added backticks.